platform/features/plot/src/SubPlot.js

```diff
@@ -64,6 +64,16 @@ define(
             this.updateTicks();
         }
 
+        /**
+         * Tests whether this subplot has domain data to show for the current pan/zoom level. Absence of domain data
+         * implies that there is no range data displayed either
+         * @returns {boolean} true if domain data exists for the current pan/zoom level
+         */
```

Comment:
I'd mark this as `@private` since it's not used from anywhere outside of SubPlot